README.md

```diff
@@ -69,158 +68,131 @@ B) **Rendering in Twig** is shown below in the [Rendering Pagerfantas](#renderin
 
 ```twig
-<div class="pagerfanta">
-    {{ pagerfanta(my_other_pager, 'default', {'pageParameter': '[page_other]'}) }}
-</div>
+{{ pagerfanta(my_pager, 'default', {'pageParameter': '[other_page]'}) }}
 ```
 
-Note the square brackets around `page_other`; this won't work without them.
+Note that the page parameter *MUST* be wrapped in brackets (i.e. `[other_page]`).
 
-### Twitter Bootstrap
+See the [Pagerfanta documentation](https://github.com/whiteoctober/Pagerfanta) for the list of supported options.
 
-The bundle has support for views using Twitter Bootstrap.
+## Available Views
 
-For Bootstrap 2:
+### Default Views
 
-```twig
-<div class="pagerfanta">
-    {{ pagerfanta(my_pager, 'twitter_bootstrap') }}
-</div>
-```
+All of the views provided in the `pagerfanta/pagerfanta` are available by default for use with this bundle.
 
-For Bootstrap 3:
+The below table lists the view names and the corresponding class. 
 
-```twig
-<div class="pagerfanta">
-    {{ pagerfanta(my_pager, 'twitter_bootstrap3') }}
-</div>
-```
+| View Name            | Class Name                              |
+| -------------------- | --------------------------------------- |
+| `default`            | `Pagerfanta\View\DefaultView`           |
+| `semantic_ui`        | `Pagerfanta\View\SemanticUiView`        |
+| `twitter_bootstrap`  | `Pagerfanta\View\TwitterBootstrapView`  |
+| `twitter_bootstrap3` | `Pagerfanta\View\TwitterBootstrap3View` |
+| `twitter_bootstrap4` | `Pagerfanta\View\TwitterBootstrap4View` |
 
+### Twig View
 
-For Bootstrap 4:
+_This feature was introduced in BabDevPagerfantaBundle 2.2_
 
-```twig
-<div class="pagerfanta">
-    {{ pagerfanta(my_pager, 'twitter_bootstrap4') }}
-</div>
-```
+This bundle provides a Pagerfanta view which renders a Twig template.
 
-### Semantic UI
+The below table lists the available templates and the CSS framework they correspond to.
 
-The bundle also has a Semantic UI view.
+| Template Name                                          | Framework                                            |
+| ------------------------------------------------------ | ---------------------------------------------------- |
+| `@BabDevPagerfantaBundle/default.html.twig`            | None (Pagerfanta's default view)                     |
+| `@BabDevPagerfantaBundle/semantic_ui.html.twig`        | [Semantic UI](https://semantic-ui.com) (version 2.x) |
+| `@BabDevPagerfantaBundle/twitter_bootstrap.html.twig`  | [Bootstrap](https://getbootstrap.com) (version 2.x)  |
+| `@BabDevPagerfantaBundle/twitter_bootstrap3.html.twig` | [Bootstrap](https://getbootstrap.com) (version 3.x)  |
+| `@BabDevPagerfantaBundle/twitter_bootstrap4.html.twig` | [Bootstrap](https://getbootstrap.com) (version 4.x)  |
 
-```twig
-<div class="pagerfanta">
-    {{ pagerfanta(my_pager, 'semantic_ui') }}
-</div>
-```
+*NOTE* The Twig view ignores the `prev_message` and `next_message` Pagerfanta options in favor of translations directly in the templates.
```

Comment:
I get that it doesn't make an awful lot of sense, but it might still be convenient to just pass this option if one wants to use a distinct message in just that one place only. Why not allow this option for twig view as well, without running the message received this way through the translator?

Reply:
Still on my "think about it" list.  It was 3 AM by the time I had gotten to a "this is good enough" state and didn't even think to go through all the options that the default views expose.